compute/cker/include/cker/operation/ReLU6.h

```diff
@@ -29,25 +29,12 @@ namespace nnfw
 namespace cker
 {
 
-inline void ReLU6(const Shape &input_shape, const float *input_data, float *output_data)
+inline void ReLU6(const Shape &input_shape, const float *input_data, const Shape &output_shape,
```

Comment:
(Optional)
I'm not sure if input_shape and output_shape always have the same shape.
How about remove `output_shape` from parameters or adding to check both shapes?

Reply:
@ragmani 

Oh, I will add assertion to check their shape. Thank you :)